packages/gestalt/src/Link.css

```diff
@@ -16,3 +16,30 @@
 .block {
   composes: block from "./Layout.css";
 }
+
+.tab {
+  composes: pointer from "./Cursor.css";
+  user-select: none;
+}
+
+.tab:focus {
+  position: relative;
+}
+
+.tabIsNotActive {
+  composes: whiteBg from "./Colors.css";
+}
+
+.tabIsNotActive:focus,
+.tabIsNotActive:hover {
+  background-color: #efefef;
```

Comment:
It was `#f1f1f1`, but it's not one of our standard colors. I'm changing it to the closest standard color.